public/js/profileList.js

```diff
@@ -0,0 +1,57 @@
+const openEditProfileForm = (profile) => {
+  // eslint-disable-next-line no-underscore-dangle
+  window.location.href = `${window.location.origin}/views/profileForm.html?profileId=${profile._id}`;
+};
+
+const openNewProfileForm = () => {
+  window.location.href = `${window.location.origin}/views/profileForm.html`;
+};
+
+window.onload = () => {
+  const navButton = document.getElementById('profilesNav');
+  const tableContent = document.getElementById('table-content');
+  const addProfileButton = document.getElementById('addProfile');
+
+  const deleteProfile = (id, name, event) => {
+    event.stopPropagation();
+    document.getElementById('modal-title').innerText = 'You are about to delete a Professional Profile:';
+    document.getElementById('modal-data-inputs').innerText = `Profile: ${name}`;
+    // eslint-disable-next-line no-undef
+    openModal();
+    // eslint-disable-next-line no-undef
+    document.getElementById('cancel-button').onclick = closeModal;
+    document.getElementById('confirm-button').onclick = () => {
+      const url = `${window.location.origin}/api/profiles/${id}`;
+      fetch(url, {
+        method: 'DELETE',
+        headers: {
+          'Content-type': 'application/json',
+        },
+      })
+        .then(window.location.reload())
```

Comment:
you should check if the deletion was ok before redirecting the page. Anyway, you should close the modal and get the new profiles list instead of redirecting the page.

Reply:
Fixed it!